jvm/src/test/scala/tastyquery/ReadTreeSuite.scala

```diff
@@ -463,7 +477,9 @@ class ReadTreeSuite extends BaseUnpicklingSuite(withClasses = false, withStdLib
     assert(containsSubtree(intFieldMatch)(clue(tree)))
   }
 
-  testUnpickle("object", simple_trees / tname"ScalaObject") { tree =>
+  testUnpickleTopLevel("object", simple_trees / tname"ScalaObject") { tree =>
+    // FIXME This test should not need `testUnpickleTopLevel`
```

Comment:
I guess this is a limitation of requiring a TopLevelDeclPath - you can only inspect classes

Reply:
No it's a problem with the classloading. The top-level module symbols don't know how to trigger the unpickling for the file. There's even a comment:
https://github.com/scalacenter/tasty-query/blob/435a4f540c0340bdc83af5a6ada66d8f74ae705f/shared/src/main/scala/tastyquery/reader/classfiles/Classpaths.scala#L205

Reply:
oh yeah I remember I did that now - ok we can look at this another time :)